source/renderer/app/components/sidebar/types.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import type { SidebarWalletType } from '../../types/sidebarTypes';
+
+export type SidebarMenus = {
+  wallets: ?{
```

Comment:
Why walletS ? is not an array as far I can see. Beside that, would not be better to have a dedicated type for wallets? Just not sure if the typescript migration tool understands it :)

Reply:
This was actually just moved from sidebar file to improve readability. The type is the same as before, not sure if it should be plural or not.

https://github.com/input-output-hk/daedalus/blob/91bcf96c8eafa22843e71abb0ebce2b72bf672a9/source/renderer/app/components/sidebar/Sidebar.js#L28